src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageSelectorModel.cs

```diff
@@ -2,25 +2,30 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Diagnostics;
 using System.IO;
 using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Primitives;
 
 namespace Microsoft.AspNetCore.Mvc.RazorPages.Internal
 {
     public static class PageSelectorModel
     {
         private const string IndexFileName = "Index.cshtml";
 
-        public static void PopulateDefaults(PageRouteModel model, string routeTemplate)
+        public static void PopulateDefaults(PageRouteModel model, string pageRoute, string routeTemplate)
```

Comment:
Breaking change?

Reply:
It's an internal type. Don't think ApiCheck complained.

Reply:
Ah, yes. I believe this is OK in non-patch releases.